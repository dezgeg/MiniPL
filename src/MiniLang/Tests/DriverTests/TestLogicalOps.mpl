print "Truth table for NOT\n";

print "NOT false: ";
print !false;
print "\n";

print "NOT true: ";
print !true;
print "\n";

print "\nTruth table for AND\n";

print "false AND false: ";
print false & false;
print "\n";

print "false AND true: ";
print false & true;
print "\n";

print "true AND false: ";
print true & false;
print "\n";

print "true AND true: ";
print true & true;
print "\n";

