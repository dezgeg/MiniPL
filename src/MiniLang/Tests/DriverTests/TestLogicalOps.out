Truth table for NOT
NOT false: true
NOT true: false

Truth table for AND
false AND false: false
false AND true: false
true AND false: false
true AND true: true
